pgcommitfest/commitfest/admin.py

```diff
@@ -1,19 +1,22 @@
 from django.contrib import admin
-
-from .models import (
-    CfbotBranch,
-    CfbotTask,
-    CommitFest,
-    Committer,
-    MailThread,
-    MailThreadAttachment,
-    Patch,
-    PatchHistory,
-    PatchOnCommitFest,
-    TargetVersion,
-    Topic,
-)
-
+from django.utils.timezone import now
+from .models import *
+
+class CommitfestAdmin(admin.ModelAdmin):
+    @admin.action(description="Start selected Commitfest")
+    def startCommitfest(self,request,queryset):
```

Comment:
Instead of integrating these in the admin panel I think it would feel more natural to put these buttons next to the admin buttons:

![image](https://github.com/user-attachments/assets/e9d8b9d1-ee36-4014-b93f-dbabd8a184cc)

The "Start commitfest" button should also do something else important: Change the next "Future" commitfest to "Open", or if no "Future" commitfest exist, create a new "open" commitfest.